src/test/scala/uk/gov/hmrc/play/views/layouts/TestWithoutPlayAppSpec.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.play.views.layouts
+
+import akka.util.Timeout
+import org.scalatest.{Matchers, WordSpec}
+import play.api.mvc.Result
+import play.api.test.{DefaultAwaitTimeout, FakeRequest, FutureAwaits}
+import uk.gov.hmrc.play.views.layouts.test.{TestAssetsConfig, TestOptimizelyConfig, TestWithoutPlayAppController}
+import uk.gov.hmrc.test.AkkaMaterializerSpec
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+class TestWithoutPlayAppSpec extends WordSpec with Matchers with FutureAwaits with DefaultAwaitTimeout with AkkaMaterializerSpec {
+
+  override implicit val defaultAwaitTimeout: Timeout = 5 seconds
+
+  "templates that use Play configuration (head and footer)" should {
```

Comment:
just a note on the intention of this code - the intention is that if somebody makes another template need configuration they should add it to this test, to make sure that all templates can be rendered without a started Play app

so another way to word this (for explanation, I'm not suggesting we actually put this in the code) is "templates that use Play configuration, which at the time of writing happen to be head and footer but which could include other templates in future"